templates/typescript-app/package.json

```diff
@@ -8,11 +8,14 @@
   "scripts": {
     "import": "cdk8s import",
     "synth": "cdk8s synth",
-    "compile": "./node_modules/typescript/bin/tsc",
-    "watch": "./node_modules/typescript/bin/tsc -w",
+    "compile": "tsc --build",
+    "watch": "tsc --build -w",
     "test": "jest",
     "build": "npm run compile && npm run test && npm run synth",
     "upgrade": "npm i cdk8s@latest cdk8s-cli@latest",
     "upgrade:next": "npm i cdk8s@next cdk8s-cli@next"
+  },
+  "devDependencies": {
+    "typescript": "~4.7.4"
```

Comment:
This actually isn't need. We currently have a somewhat round about way of installing dev dependencies in the templates:

https://github.com/cdk8s-team/cdk8s-cli/blob/2e590cff9c14cad92beed9a623512852d4de5a13/templates/typescript-app/.hooks.sscaff.js#L16-L22

So we if just remove this we should be good. 

```suggestion
```